src/main/java/dev/snowdrop/release/App.java

```diff
@@ -245,17 +245,22 @@ public void setupRelease(
                 LOG.infof("Release ticket %s already exists, skipping cloning step", releaseTicket);
             } catch (Exception e) {
                 // if we got an exception, assume that it's because we didn't find the ticket
-                issue = clone(release, token);
+                if (!release.isTestMode()) {
+                    issue = clone(release, token);
+                }
             }
         } else {
             // no release ticket was specified, clone
-            issue = clone(release, token);
-
+            if (!release.isTestMode()) {
+                issue = clone(release, token);
```

Comment:
I'm just wondering if we could add the flag to the `clone` method signature and avoid the 3 `if` here. wdyt?

Reply:
done